src/main/java/twilightforest/entity/boss/EntityTFIceCrystal.java

```diff
@@ -19,9 +20,14 @@
 	private int crystalAge;
 	private int maxCrystalAge = -1;
 
+	public EntityTFIceCrystal(World worldIn) {
+		super(TFEntities.ice_crystal.get(), worldIn);
+	}
+
 	public EntityTFIceCrystal(EntityType<? extends EntityTFIceCrystal> type, World world) {
 		super(type, world);
 
+		// TODO: ???
```

Comment:
if it was already commented in 1.12, remove it. otherwise it's just a setHeldItem call

Reply:
Ah, yeah, blame says it's been commented out for five years.